### DISCUSSION TITLE
Structured data footnotes

### DISCUSSION THREAD

**Discussion by kyledenlinger** [knowledgefutures/pubpub#2212] [Support]
Here's what I'm doing in case someone can reproduce. I'm going about this in a Very Wrong Way but there's a bug in here somewhere regardless.

1.  Upload a docx containing footnotes. Created in Word, footnotes inserted with Zotero, Chicago 17 full note, field codes stripped before uploading.  Can provide file upon request; it's student work, so not linking here.
2. Footnotes appear as nothing more than (Author, Date) stored as rich text. No bueno.
3. I enter edit mode for TextNote1, delete the rich text, and quick-copy bibtex from Zotero into structured data. Click "Update footnote." 
4. This fixes it for TextNote1, but now TextNote2 disappears from the list, but not the text. The list is now incorrectly renumbered -- TextNote2 is ignored, ListNote2 is actually TextNote3.
5. I enter edit mode for TextNote2. Delete the rich text, drag bibtex, Update Footnote. TextNote2 displays correctly on hover, but ListNote2 is still pointing to TextNote3. TextNote2 still does not appear in the list.
6. I enter edit mode for TextNote2. Bibtex is still in Structured Data, but if I put _anything_ in Rich Text (a space, e.g.), TextNote2 displays correctly in the list, but now with a trailing space.

Hopefully that's not totally inscrutable.

**Comment by kyledenlinger** [comment]
Here's the latest release of the pub in question if you want to look through recent revision history. https://decarbonizingcharacter.pubpub.org/pub/prudent-decisions/release/3

**Reply by gabestein** [reply]
Thanks Kyle, and sorry you're dealing with this. Can you send the file to hello@pubpub.org so we can take a look? My guess is it's related to zotero + footnotes, which I don't think is well supported by pandoc, which we're using for importing.

**Reply by kyledenlinger** [reply]
Thanks, @gabestein. I managed. 

I'm not sure it's a Zotero issue -- I stripped the Zotero field codes from the document before uploading. Does pandoc handle footnotes and citations that are generated entirely with Word's footnote and/or citation features, not using Zotero at all?

I'd really love some more documentation on importing footnotes/citations and bibliographies. E.g., I know I can upload and designate a  bibtex file as a bibliography, but what does that do, and does that bibliography connect to in-text references, and if so, how? Admittedly I haven't played around with it very much but I want to get into the practice of using structured data instead of rich text whenever possible, and citation is always a stumbling block when it comes to importing pubs authored elsewhere.

**Reply by gabestein** [reply]
Thanks Kyle. We should absolutely improve documentation. For example, I don't actually think a .bib file does anything when you're importing from Word.

My current understanding is PubPub, via Pandoc, should handle Zotero, Mendeley, and Endnote citations embedded in Word docs (not footnotes), and not built-in Word citations. But it should handle Word footnotes as text. 

Anyhow, if you get a chance, please do you send in the doc in question we can take a look and see what's going on.

**Comment by kyledenlinger** [comment]
I can't be sure whether this is related, but I learned today when dealing with a similar issue that if I insert a footnote into a pub at the end of a line but immediately before an image, the footnote appears in the text just fine, but it is ignored in the footnotes list at the end of the pub, throwing off the numbering as a result. The footnote will only appear in the list after forcing a line break immediately after the in-text reference and before the image.
